test/react-native-cli/features/fixtures/Dockerfile

```diff
@@ -10,5 +10,6 @@ COPY . /app
 WORKDIR /app
 
 RUN npm i -g bugsnag-react-native-cli-*.tgz
+RUN npm i
```

Comment:
Without this, the project dependencies get installed as a side effect of installing `@bugsnag/source-maps` or `@bugsnag/react-native` during the tests. Since the new fixture has more dependencies, it caused the tests to time out. This means the project deps are installed at build time rather than during the tests, which should also speed up the other tests due to the build being cache-able.